lib/ansible/modules/database/influxdb/influxdb_retention_policy.py

```diff
@@ -100,13 +107,11 @@
 
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.influxdb import InfluxDb
-from ansible.module_utils._text import to_native
```

Comment:
keep this but...

Reply:
@theo-bot Then maybe you should rebase or update the branch to rule out these changes?